packages/vite/src/node/plugins/css.ts

```diff
@@ -246,7 +246,7 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
         return
       }
 
-      const modules = cssModulesCache.get(config)!.get(id)
+      const modules = cssModulesCache.get(config) && cssModulesCache.get(config)!.get(id)
```

Comment:
Why written so complex?
Just do

```suggestion
      const modules = cssModulesCache.get(config)?.get(id)
```

Reply:
updated.